language-api/src/main/java/de/jplag/tokentypes/ObjectOrientationTokens.java

```diff
@@ -0,0 +1,22 @@
+package de.jplag.tokentypes;
+
+import de.jplag.TokenType;
+
+public enum ObjectOrientationTokens implements TokenType {
+    CONSTRUCTOR("CONST"),
+    CONSTRUCTOR_END("CONT_END"),
+    METHOD("METHOD"),
+    METHOD_END("METHOD_END"),
```

Comment:
Are methods really that object-oriented? I would say no, as this token would correspond to any function/routine.